sql/src/main/java/io/crate/metadata/ColumnIdent.java

```diff
@@ -72,7 +72,7 @@ public ColumnIdent(String name, String childName) {
 
     public ColumnIdent(String name, @Nullable List<String> path) {
         this.name = name;
-        this.path = MoreObjects.firstNonNull(path, ImmutableList.of());
+        this.path = MoreObjects.firstNonNull(path, ImmutableList.<String>of());
```

Comment:
@mfussenegger funnily it would not have compiled otherwise ...

```
.../sql/src/main/java/io/crate/metadata/ColumnIdent.java:75: error: incompatible types: inferred type does not conform to upper bound(s)
        this.path = MoreObjects.firstNonNull(path, ImmutableList.of());
                                            ^
    inferred: List<? extends Object>
    upper bound(s): List<String>,Object
```